src/components/Main.js

```diff
@@ -4,17 +4,20 @@ import './Main.css';
 import SideMenu from './SideMenu';
 import UserInfo from './UserInfo';
 
-const Main = () => {
+const Main = ({isFetching, user}) => {
+  if (isFetching) {
+    return <p>Fetching...</p>
+  }
   return (
-    <>
+    <div data-testid='main'>
```

Comment:
В розмітці бажано використовувати подвійні лапки 
```suggestion
    <div data-testid="main">
```

Reply:
Fixed with prettifier